front-end/src/components/main-page/MainPage.jsx

```diff
@@ -78,12 +102,25 @@ class MainPage extends Component {
                 }}
               />
             </div>
-            <div className="justify-center flex">
+            <div className="justify-center flex flex-col sm:flex-row">
+              <div className="absolute py-32">
+                <input
+                  className="bg-off-white text-xl rounded-md px-4 py-2 drop-shadow-md outline-none transition ease-in-out
+                  border border-solid border-purple-100
+                  hover:border-purple-200
+                  focus:border-purple-300 focus:ring-purple-300"
+                  aria-label="Enter Project ID"
+                  placeholder="Enter Project ID"
+                  onChange={this.handleChange}
+                  value={this.state.inputValue}
+                  title="Each transcript has a unique Project ID that you can enter to specify which analysis to view."
+                />
+              </div>      
               <BaseButton
                 click={this.openAnalysisPage}
                 text="View Analysis"
                 size="lg"
-                isDisabled={!isTranscriptUploaded}
+                isDisabled={this.isInputBlank()}
```

Comment:
Shouldnt this also check for whether there is a transcript uploaded as well? Maybe this.isInputBlank || !isTranscriptUploaded (or some equivalent to it)

Reply:
Good call